spec/har/page_spec.rb

```diff
@@ -18,7 +18,20 @@ module HAR
       it "has a PageTimings instance" do
         page.timings.should be_kind_of(PageTimings)
       end
-    end
 
+      describe "#entries_before" do
+        it "requires a DateTime" do
+          expect { page.entries_before("a string") }.to raise_error(TypeError)
+        end
+
+        it "filters entries that responded before the specified time" do
+          time = page.started_date_time + 0.250
+          entries = page.entries_before(time)
+          entries.each do |entry|
+            (entry.started_date_time + entry.time / 1000.to_f).should < time
```

Comment:
Could we use `expect` instead of `should` here?

```ruby
expect(entry.started_date_time + entry.time / 1000.to_f).to be < time
```

Reply:
We can, but I was going with consistency with the current style using RSpec 2.0, figuring a larger task would be to bump to 3.8 and update specs in general to the more modern matcher style.